addons/account_edi_ubl_bis3/models/account_edi_format.py

```diff
@@ -102,6 +102,22 @@ def grouping_key_generator(tax_values):
 
         return values
 
+    ####################################################
+    # Account.edi.format override
+    ####################################################
+
+    def _check_move_configuration(self, invoice):
+        errors = super()._check_move_configuration(invoice)
+        edi_format = invoice.journal_id.edi_format_ids
+        edi_format_required = any(ef._is_required_for_invoice(invoice) for ef in edi_format)
+        is_edi_needed = invoice.is_invoice(include_receipts=False) and edi_format_required
+        if is_edi_needed:
+            customer = invoice.commercial_partner_id
+            supplier = invoice.company_id.partner_id.commercial_partner_id
+            if supplier.country_code in COUNTRY_EAS and customer.country_code not in COUNTRY_EAS:
+                errors.append(_("The customer's country must be specified and be part of the EAS (Electronic Address Scheme) code list."))
```

Comment:
In stable version, you need to add this term into the `.pot` file for translations